geopackage_validator/validations/geometry_type_check.py

```diff
@@ -41,26 +70,44 @@ def query_gpkg_metadata_geometry_types(dataset):
     dataset.ReleaseResultSet(geometry_type_names)
 
 
+def aggregate(results):
+    aggregate = {}
+
+    for layer_name, geom_type, feature_id, geometry_expected in results:
+        key = (layer_name, geom_type).__hash__()
+
+        if key in aggregate:
+            aggregate[key]["amount"] += 1
+            aggregate[key]["desc"] = "times, example record id's"
+            if aggregate[key]["amount"] <= 5:
+                aggregate[key]["rowid_list"] += [feature_id]
```

Comment:
Dit lijstje met `feature_ids` is een mooie feature zo en kost niet zoveel omdat je er toch wel overheen loopt. Ding is alleen dat je, mocht het handig zijn voor een nieuwe op rijen validatie om een keer via SQL een query te doen, je hier dan mogelijk ook de verwachting schept dat je daar ook een lijstje met feature_ids terug gaat geven. En in die case kan dit weleens complex / duur worden.